packages/wdio-browserstack-service/src/util.ts

```diff
@@ -207,16 +209,18 @@ export async function nodeRequest(requestType: string, apiEndpoint: string, opti
         clearTimeout(timeoutId)
 
         return await response.json()
-    } catch (error : any) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    } catch (error: any) {
         BStackLogger.debug(`Error in firing request ${apiUrl}/${apiEndpoint}: ${format(error)}`)
         const isLogUpload = apiEndpoint === UPLOAD_LOGS_ENDPOINT
         if (error && error.response) {
             const errorMessageJson = error.response.body ? JSON.parse(error.response.body.toString()) : null
             const errorMessage = errorMessageJson ? errorMessageJson.message : null
-            if (errorMessage) {
-                isLogUpload ? BStackLogger.debug(`${errorMessage} - ${error.stack}`) : BStackLogger.error(`${errorMessage} - ${error.stack}`)
+            const logMessage = errorMessage ? `${errorMessage} - ${error.stack}` : error.stack
```

Comment:
This logic has been changed, please verify if this new behaviour is correct (just a double check as it is changed)

Reply:
good catch!